src/commands/reminders/remind.ts

```diff
@@ -47,7 +47,8 @@ export default class extends Command {
 		const reminder: Reminder = {
 			owner: interaction.user.id,
 			content,
-			mode: interaction.channel.type === 'DM' ? 'private' : 'public',
+			// mode: interaction.channel.type === 'DM' ? 'private' : 'public',
+			mode: 'public', // temporary
```

Comment:
Make sure to keep an eye on the temporary stuff; this comment is mostly a footnote for myself

Reply:
i think this was a problem with interactions since reminders can't be set in DM (because commands can't be run in private), but we can definitely take a second look at this at some point